service/chartconfig/v1/helm/helm_test.go

```diff
@@ -321,3 +321,64 @@ func Test_InstallRelease(t *testing.T) {
 		})
 	}
 }
+
+func Test_UpdateRelease(t *testing.T) {
```

Comment:
I think this should be `Test_UpdateReleaseFromTarball`.

Reply:
Sure, done